### PR TITLE
Add scripts for Postgres Version 15.

### PR DESCRIPTION
**Pull request by rakibulhossain**
Signed-off-by: Rakibul-Hossain <rakibul.hossain@appscode.com>

### CODE REVIEW

**Review by heheh13 — APPROVED** [review]
LGTM!

File: role_scripts/15/primary/postgresql.conf [hunk 1]
```diff
@@ -0,0 +1,809 @@
+# -----------------------------
```
Comment (Emon46) [inline role_scripts/15/primary/postgresql.conf:1]: update this file with latest config file by postgres https://github.com/postgres/postgres/blob/REL_15_STABLE/src/backend/utils/misc/postgresql.conf.sample

File: role_scripts/15/standby/run.sh [hunk 2]
```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bash
```
Comment (Emon46) [inline role_scripts/15/standby/run.sh:1]: update this file with latest config file by postgres 15 https://github.com/postgres/postgres/blob/REL_15_STABLE/src/backend/utils/misc/postgresql.conf.sample

**Review by Emon46 — CHANGES_REQUESTED** [review]

File: role_scripts/15/primary/postgresql.conf (hunk 1 above)
Comment (rakibulhossain) [reply role_scripts/15/primary/postgresql.conf:1]: I brought it from the master. will it work?

**Review by Emon46 — APPROVED** [review]
LGTM!

**Comment by tamalsaha** [comment]
Config source:
https://github.com/postgres/postgres/blob/REL_15_0/src/backend/libpq/pg_hba.conf.sample

**Pull request merged**